apps/prairielearn/src/migrations/20240308203938_workspaces__disk_usage__add.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE workspaces
+ADD COLUMN disk_usage BIGINT;
```

Comment:
In #7170, I had proposed also recording a count of all files. I'm inclined to think that's not necessary; thoughts?

Reply:
I agree that this might in theory be interesting but is really not very important for us in practice.

Reply:
Consider naming the column something like `disk_usage_bytes` to encode the unit.

Reply:
Changed in 2a92b4b.